lib/helpers.js

```diff
@@ -80,8 +80,14 @@ export function visitMessage(message: LinterMessage, reference: boolean = false)
   let messageFile
   let messagePosition
   if (reference) {
-    invariant(message.version === 2, 'Only messages v2 are allowed in jump to reference')
-    invariant(message.reference && message.reference.file, 'Message.reference must be valid')
+    if (message.version !== 2) {
+      console.warn('[Linter-UI-Default] Only messages v2 are allowed in jump to reference. Ignoring')
+      return
+    }
+    if (!message.reference || !message.reference.file) {
+      console.warn('[Linter-UI-Default] Message does not have a valid reference. Ignoring')
```

Comment:
Do we really need to warn the user if the (mis)click on a message with no reference? Is there a visual indication that a message has a reference?